src/components/cycle-section/cycle-section/CycleSection.module.scss

```diff
@@ -0,0 +1,111 @@
+.panelContainer {
```

Comment:
I think that it would be nice to store colors in variables

Reply:
Done!